dace/serialize.py

```diff
@@ -120,7 +139,12 @@ def from_json(obj, context=None, known_type=None):
                         known_type.__name__)
 
     if t:
-        return _DACE_SERIALIZE_TYPES[t].from_json(obj, context=context)
+        deserialized = _DACE_SERIALIZE_TYPES[t].from_json(obj, context=context)
+        if deserialized is None:
```

Comment:
hmm what if the value really is None? maybe we should look for raised exceptions? (for example, KeyError in _DACE_SERIALIZE_TYPES would also be supported etc.)